zap/src/main/java/org/parosproxy/paros/core/scanner/Scanner.java

```diff
@@ -325,6 +294,21 @@ public void scan(Target target) {
         }
     }
 
+	private HostProcess getAndInitHostProcess(String hostAndPort, StructuralNode node) {
```

Comment:
`createHostProcess`

Needs a ZAP comment after the license header.

Reply:
Hi, so I need to add `// ZAP: 2020/05/19 Remove HostProcess code duplication` after the `// ZAP: 2019/11/09 Ability to filter to active scan (Issue 5278)` ?

Reply:
That works.